tests/test_small_output.run

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
```

Comment:
I'd like to avoid adding new sh tests and go towards something that'll run more easily on Windows (#82).  Specifically here I would just put them inside the C test file and run that directly.